website/templates/comments2.html

```diff
@@ -45,7 +45,7 @@ <h2 class="text-xl lg:text-3xl font-bold text-gray-900 ">Comments ({{ all_commen
                                         src="{% static 'images/dummy-user.png' %}"
                                         alt="User profile">
                                 {% endif %}
-                                {{ comment.author }} {{ comment.author_url }}
+                                {{ comment.author }}is
```

Comment:
Is this right?

Reply:
Yes I just added it for testing but forgot to remove it

Reply:
Should there be a space between "is" ?

Reply:
OO, It shouldn't be there. Thanks i removed it